distrib/starlake.cmd

```diff
@@ -57,6 +57,39 @@ if "%~1"=="install" (
 
 goto :eof
 
+:add_server_cert_to_java_keystore
+  if defined JAVA_HOME (
+    if defined SL_INSECURE (
+        openssl s_client -proxy %proxy% -showcerts -servername %~1 -connect %~1:443 > %~1.tmp 2>nul
+    ) else if defined CA_PATH (
+        echo Verifying %~1 certificate using CApath %CA_PATH%...
+        openssl s_client -CApath %CA_PATH% -proxy %proxy% -verify_return_error -showcerts -servername %~1 -connect %~1:443 > %~1.tmp 2>nul
+    ) else if DEFINED CA_FILE (
+        echo Verifying %~1 certificate using CAfile %CA_FILE%...
+        openssl s_client -CAfile %CA_FILE% -proxy %proxy% -verify_return_error -showcerts -servername %~1 -connect %~1:443 > %~1.tmp 2>nul
+    ) else (
+        echo Retrieving %~1 certificate...
```

Comment:
Why ...

Reply:
because in this case, SL_INSECURE has not been defined

Reply:
Why do we need to call openssl when SL_INSECURE is not defined ?